dask/dataframe/core.py

```diff
@@ -3354,7 +3355,7 @@ class Series(_Frame):
     _partition_type = pd.Series
     _is_partition_type = staticmethod(is_series_like)
     _token_prefix = "series-"
-    _accessors = set()
+    _accessors: Set[Any] = set()
```

Comment:
```suggestion
    _accessors: ClassVar[set[str]] = set()
```
There is no benefit in using an abstract type here.